startup/30-user.py

```diff
@@ -760,16 +770,17 @@ def series(det='eiger4m',shutter_mode='single',expt=.1,acqp='auto',imnum=5,comme
     if use_xbpm:
         caput( 'XF:11IDB-BI{XBPM:02}FaSoftTrig-SP',1,wait=True) #yugang add at Sep 13, 2017 for test fast shutter by using xbpm
         print('Use XBPM to monitor beam intensity.')
-    if feedback_on:
-        RE(prep_series_feedback())
+
     #RE(count([detector]),Measurement=comment)  ### ACQUISITION
     if PV_trigger:
-      dets = detlist      
-      RE(wait_for_pv(dets,trigger_PV,md=RE.md),Measurement=comment)
+      dets = detlist     
+      RE(wait_for_pv(dets,trigger_PV,feedback_on=feedback_on,md=RE.md),Measurement=comment)
```

Comment:
This whole function should be turned into a plan.  Basically `RE(...)` -> `yield from ...`.